lib/filters/global/parsoid.js

```diff
@@ -109,6 +130,15 @@ module.exports = function (conf) {
             },
             '/v1/{domain}/pages/{key}/data-parsoid/{revision}': {
                 get: { request_handler: getDataParsoidRevision }
+            },
+            '/v1/{domain}/transform/{from}/to/{to}': {
+                post: { request_handler: parsoidConvert(conf.parsoidHost) }
+            },
+            '/v1/{domain}/transform/{from}/to/{to}/{title}': {
```

Comment:
We should eventually narrow this down to html/to/wikitext and perhaps wikitext/to/html.